doc/packages/emacs.section.md

```diff
@@ -1,8 +1,16 @@
 # Emacs {#sec-emacs}
 
+[Emacs](https://www.gnu.org/software/emacs/) is the advanced, extensible, customizable, self-documenting editor.
+
+At its core is an interpreter for Emacs Lisp (shortly, Elisp), a dialect of the Lisp programming language with extensions to support text editing.
+
 ## Configuring Emacs {#sec-emacs-config}
 
-The Emacs package comes with some extra helpers to make it easier to configure. `emacs.pkgs.withPackages` allows you to manage packages from ELPA. This means that you will not have to install that packages from within Emacs. For instance, if you wanted to use `company` `counsel`, `flycheck`, `ivy`, `magit`, `projectile`, and `use-package` you could use this as a `~/.config/nixpkgs/config.nix` override:
+Nixpkgs provides a framework that leverages Emacs via Nix.
+
+`emacs.pkgs.withPackages` allows managing packages from various Elisp package repositories (ELPA, MELPA etc.) and even third-party stand-alone packages from Nix, without relying on Emacs to download and install them.
```

Comment:
"ELPA" is Emacs Lisp Package Archive.

https://elpa.gnu.org/ is GNU Emacs Lisp Package Archive (or GNU ELPA), an ELPA run by GNU.

Maybe we can use more precise words here.

Reply:
I will remove this commit.